src/home/swap-container/components/swap/index.tsx

```diff
@@ -363,15 +367,21 @@ const Swap = ({
         ),
       });
       trackEvent('DCA - Create position submitting');
-      const result = await positionService.deposit(
+      // change it to xCall here.
+      // eslint-disable-next-line no-console
```

Comment:
please don't disable these lint-rules, its fine if it fails the lint action since later you can remove them

Reply:
Noted ✅